printer_mods/Baltojikale/LoudOwl_aka_Stabby/README.md

```diff
@@ -0,0 +1,42 @@
+# LoudOwl aka Stabby
+
+<img src="Visual/LoudOwl_2.jpg" width="400px" />
+<img src="Visual/LoudOWL.jpg" width="400px" />
+
+This is alternative part cooling fan solution which uses dual 5015 fan setup for Afterburner Toolhead. Replaces original parts that house 4020 fan or any other similar type of mod. This mod is more of situational add on for PLA or similar type filaments which might require a lot of cooling (or printing at super high speeds). It is not constant solution as it increases gantry weight.
+
+## Hardware
+* 2x 5015 24v fans
+* 1x M3x20 SCHS (DIN912)
+
+## Printing
+
+There are two sets of STLs. One with inbuild supports and one without (for those who trust their slicers autosupports). Tested with recommended settings for Voron parts: 4 perimeters, 5 tops/bottoms, 40% infill. Though it is possible to use less plastic to reduce weight of plastic parts: 3 perimeters, 3 tops bottoms, 15-25% infill. 
+Both versions of STL were tested by printing and assembling !!!
+
+<img src="Visual/Supp_removal.jpg" width="400px" />
```

Comment:
That looks really really messy after support removal, and it seem that it would be difficult to remove the supports.  Does it need that much support?  Some bridging is acceptable if it's in a straight line.  It may even require less supports if you print it front side down.

Alternatively you could separate the two halves as their own parts that can print without needing supports at all, and then screwed together.

Reply:
Ok thanks for a good tip. Additional 0.2mm extrusion in those places autofixed bridging orientation. here pict of iterations https://drive.google.com/file/d/1UqKQQf3MKKV-7ycwiOXaRszQLywNhRAP/view?usp=sharing



Current design requires that much support due to fact that fans are at angle. Though there might be zones where I overdid with supports. Printing front side down will leave too much artifacts. Seen face part printed in vertical orientation that could be a solution though print time would be longer. Non the less support removal is easy.  Supports were iterated and print tested. Submitted one was 3rd iteration. Photo with explanation: https://drive.google.com/file/d/1hgGC71vPr6cmJcIECtgezaYyaEcpRqAN/view?usp=sharing

For current design I don't want to change angled fan orientation as it makes toolhead look cool. SO "Beauty requires sacrifice".
Later on planning to release flat version, much more revised to be printed support-less or with minimal support amount